### DISCUSSION TITLE
v2.6.333

### DISCUSSION THREAD

**Discussion by This-is-XiaoDeng** [Moonlark-Dev/XDbot2#116] [Announcements]
## What's Changed
* [Update] update en_laman28.json by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/77
* [todo/mov]重构 by @YifanZhuZhu in https://github.com/This-is-XiaoDeng/XDbot2/pull/78
* master by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/80
* [add]重构完成 by @YifanZhuZhu in https://github.com/This-is-XiaoDeng/XDbot2/pull/79
* 我是傻逼 by @YifanZhuZhu in https://github.com/This-is-XiaoDeng/XDbot2/pull/81
* 我 t m 就 是 个 傻 逼 by @YifanZhuZhu in https://github.com/This-is-XiaoDeng/XDbot2/pull/82
* master by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/86
* master by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/87
* master by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/88
* [fix] lang.py _lang.py by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/90
* [fix] en_laman28.json by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/91
* master by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/92
* [add] 增加了1小时、今日、总计群聊热度 by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/101
* [fix] UnboundLocalError: cannot access local variable 'data' where it is not associated with a value by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/102
* [fix] TypeError: list indices must be integers or slices, not str by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/103
* [update] 移除热度为0的群聊 by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/104
* [update] 整理代码&修复潜在bug by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/105
* [fix] RuntimeError: dictionary changed size during iteration by @montmorill in https://github.com/This-is-XiaoDeng/XDbot2/pull/106
* 大东北特供版本的zh_fzz! by @FZZkill in https://github.com/This-is-XiaoDeng/XDbot2/pull/108
* [update] en_laman28.json by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/109
* [update] pre-commit  by @xxtg666 in https://github.com/This-is-XiaoDeng/XDbot2/pull/111
* [fix] fix lang.compatibility in en_laman28.json by @Lone-Air in https://github.com/This-is-XiaoDeng/XDbot2/pull/114
* [update] quick math 群聊热度检测 by @xxtg666 in https://github.com/This-is-XiaoDeng/XDbot2/pull/115

## New Contributors
* @montmorill made their first contribution in https://github.com/This-is-XiaoDeng/XDbot2/pull/101

## 更新日志

1. 修复已知问题
2. 将setu价格调整为1.2vi
3.  添加插件：系统商城（`/shop`）、上传日志（`/upload-log`）、使用物品（`/use`）、计算器（`/calc`）、群热度排名（`/ghot`）、速算（`/quick-math`）、成就（`/achivement`）
4. 整理部分代码
5. 将背包保存任务间隔修改为 15s
6. 更新本地化语言：en_laman28、zh_cat、zh_fzz
7. 添加语言：en_laman28_reverse、en_laman28_sha256、zh_bronya、zh_null
8. 拒绝加载无法被正常加载的 lang 文件
9. 删除语言：zh_yf
10. 修改汇率算法
11. 添加物品：书与笔
12. 添加更多 tips
13. 修改随机 tips 的触发概率
14. 添加插件：`nonebot_plugin_learning_chat`（社区）
15. 将输出日志等级调整为 DEBUG
16. 为 6-count 添加纯文本排名
17. 修改本地化工具
18. 为 update 添加本地化
19. 移除了 @montmorill 
20. 为批量使用添加默认 200 的数量限制
21. 加入了 @WowStarWorld 
22. 添加成就：+0！、我爱数学
23. 为部分指令添加帮助信息
24. 为 GitHub 仓库安装 pre-commit
25. 更新了依赖列表
26. 弃用了汇率并为汇率查看添加弃用提示
27. 在回声洞被回复时向发送者发送消息 

**Full Changelog**: https://github.com/This-is-XiaoDeng/XDbot2/compare/v2.6.72...v2.6.333

<hr /><em>This discussion was created from the release <a href='https://github.com/This-is-XiaoDeng/XDbot2/releases/tag/v2.6.333'>v2.6.333</a>.</em>

**Comment by YifanZhuZhu** [comment]
愚人节快乐（

**Reply by This-is-XiaoDeng** [reply]
6

**Comment by xxtg666** [comment]
一眼望去全是傻逼（

**Comment by muderstar** [comment]
66666
